offboarding.qmd

```diff
@@ -10,46 +10,49 @@ The convention of giving two weeks notice is *not enough time to wrap up*.
 With more time, the group can help with next steps including finding new positions and creating an exit plan.
 Further, it typically takes a month or more to effectively document and transfer knowledge of what you are working on.
```

Comment:
I know this section wasn't edited in this PR, but I gotta say I don't *love* the content of this paragraph.  There are plenty of reasons someone might not want to tell their boss that they are looking for another job and there's definitely no actual requirement to disclose this, so I think I'd feel more comfortable if the wording of this was "softened" a bit.  Like, ideally everyone is comfortable discussing career moves with Kristina and sure it helps to get advanced notice, but we have processes in place to ensure that two weeks notice hopefully *is* plenty of time to transfer projects. Everyone ideally should be documenting projects as they go, not spending the month before they move on doing all the documentation on all their projects.  We have processes in place so everyone is more or less up to date on the status of all the projects we are working on, so offboarding shouldn't take a month or more.